readme.md

```diff
@@ -923,6 +923,47 @@ const HelloBundle = dynamic({
 export default () => <HelloBundle title="Dynamic Bundle" />
 ```
 
+### Custom `<App>`
+
+<p><details>
+  <summary><b>Examples</b></summary>
+  <ul><li><a href="./examples/layout-component">Using `_app.js` for layout</a></li></ul>
+  <ul><li><a href="./examples/componentdidcatch">Using `_app.js` to override `componentDidCatch`</a></li></ul>
+</details></p>
+
+- Is rendered on both client and server side
+- `this.state` should not be overwritten. Instead create a new component to handle the state ([layout example](./examples/layout-component))
```

Comment:
This is not clear. What are we going to say.

Reply:
Added a clearer example.